src/java/main/org/apache/zookeeper/server/ServerStats.java

```diff
@@ -32,6 +32,7 @@
     private long minLatency = Long.MAX_VALUE;
     private long totalLatency = 0;
     private long count = 0;
+    private long fsyncThresholdExceedCount = 0L;
```

Comment:
What do you think about using `AtomicLong` instead of synchronising every access?

Reply:
Which would also remove the overhead imposed by synchronised. Thanks for the tip, i will make the changes!